lua/impatient.lua

```diff
@@ -0,0 +1,238 @@
+-- modified version from https://github.com/lewis6991/impatient.nvim
+
+local vim = vim
+local api = vim.api
+local uv = vim.loop
+
+local get_option, set_option = api.nvim_get_option, api.nvim_set_option
+local get_runtime_file = api.nvim_get_runtime_file
+local home_dir = vim.loop.os_homedir()
+
+local impatient_start = uv.hrtime()
+local impatient_dur
+
+local M = {
+  cache = {},
+  profile = nil,
+  dirty = false,
+  path = home_dir .. "/.local/share/lunarvim/cache",
+  log = {},
+}
+
+_G.__luacache = M
+
+local mpack = require "impatient.cachepack"
+
+local function log(...)
```

Comment:
We're already having logging capabilities, see `core.log`

Reply:
I'm not sure this is worth changing, honestly. It's mostly for troubleshooting. If anything it might be worth removing any logging.

Furthermore, the current implementation in lvim uses plenary which is a plugin that is loaded later and the default logging defaults to logging to disk which can cause some overhead. It is not worth loading plenary before this just to log.